master/getting-started/bare-metal/bare-metal.md

```diff
@@ -476,14 +472,50 @@ outside the cluster:
    normal host endpoint policy).  In other words, if a host endpoint is defined
    but has no pre-DNAT policies that explicitly allow or deny a particular
    incoming packet, that packet is allowed to continue on its way, and will
-   then be accepted or dropped according to workload policy (if it is going to
-   a local workload) or to normal host endpoint policy (if not).
+   then be accepted or dropped according to normal host endpoint policy applied on
+   forward traffic or a workload policy.
+
+## Policy on forward traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from local process only. 
+
+If `applyOnForward` is `true`, host endpoint policy applies to forward traffic include:
+1. Incoming traffic from a host endpoint to a local workload (container/pod/VM).
+2. Outgoing traffic from a local workload to a host endpoint.
+3. Traffic from one host endpoint that is forward to another host endpoint.
+
+By default, `applyOnForward` is `false`. 
+
+Untracked policy and Pre-DNAT policy must have `applyOnForward` set to `true` because it applies to all forwarded traffic. 
+ 
+Policies created before calico v2.7.0 will have `applyOnForward` set to default value `false` which means host endpoint polices and implicit default deny rules no longer apply to forward traffic by default. If you want a policy to apply forwarded traffic then you must explicitly set the `applyOnForward` to `true`. 
+
+> If you upgrade from a lower version of calico to v2.7.0 and
+> want to maintain the default-deny behavior for all
+> host-endpoint forwarded traffic, you can create an empty
+> policy with `applyOnForward` set to `true` that applies to
+> all traffic on all host endpoints. 
+```
+calicoctl apply -f - <<EOF
+- apiVersion: v1
+  kind: policy
+  metadata:
+    name: empty-default-deny
+  spec:
+    types: 
+      - ingress
+      - egress
+    selector: has(host-endpoint)
+    applyOnForward: true
+EOF
+```
+> Note this policy has no `order` field specified which makes
+> it with highest order number. It will be applied after all
+> other policies.
 
 ## When do host endpoint policies apply?
 
 As stated above, normal host endpoint policies apply to traffic that arrives on
-and/or is sent to a host interface, except if that traffic comes from or is
-destined for a workload on the same host; but the rules for applying untracked
+and/or is sent to a host interface; but the rules for applying untracked
```

Comment:
- I believe `;` --> `,`
- Also, would suggest "are different in some cases" --> "differ in some cases"